Upgrade status-go to bugfix-no-messaging-rinkeby-gd04e667-12 (geth 1.7 rebase)

oskarth opened the pull request:
Includes latest status-go develop that uses geth 1.7. Here be dragons.

oskarth committed:
Upgrade status-go to develop-g90acfed (geth 1.7 rebase)

annadanchenko commented:
FIXED #2069 Android app crashes if send ether - blocker. Can't continue testing this feature on Android real device
FIXED #2089 Transactions History is empty for the account that has history of transactions
app with updated status-go has 3 known issues atm: 
FIXED #2093 No ether is received after sending request to the faucet

yenda committed:
[bug] fix #2089 and #2098 with geth update

with geth 1.7 addresses are passed with 0x prefix and mix of lower/upper cases
update the normalize hex function so that addresses are always without prefix
and entirely lower-cased.

rasom committed:
upgrade status-go

rasom commented on src/status_im/ui/screens/accounts/events.cljs:
```diff
@@ -108,13 +109,16 @@
          :as            db} :db} [_ {:keys [address] :as account} password]]
-    (let [account' (assoc account :network network
-                                  :networks networks)]
+    (let [address (utils/normalize-hex address)
```
it probably should be `utils.hex`

rasom committed:
fix util.hex alias

rasom committed:
upgrade status-go

annadanchenko commented:
#2122 Crash on login after switching network

rasom committed:
upgrade status-go

rasom committed:
upgrade status-go
